src/coreclr/src/jit/importer.cpp

```diff
@@ -13883,7 +13883,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                         bool bbIsReturn = (block->bbJumpKind == BBJ_RETURN) &&
                                           (!compIsForInlining() || (impInlineInfo->iciBlock->bbJumpKind == BBJ_RETURN));
                         LclVarDsc* const lclDsc = lvaGetDesc(lclNum);
```

Comment:
Unrelated to this PR, but this code inexplicably usees both `lclDsc` and `lvaTable[lclNum]`